main.js

```diff
@@ -28,8 +28,8 @@ const patchbay = {
 }
 
 const post = {
-  patchbay: {
-    message: bulk(__dirname, [ 'post-patchcore/message/**/*.js' ])
+  'patchbay-post': {
```

Comment:
hmm me changing this was a little superstitious .. I was having some bad problems which turned out to be me making a typo. Need to revert this